src/engine/QueryPlanner.h

```diff
@@ -403,14 +403,13 @@ class QueryPlanner {
   std::vector<SubtreePlan> applyJoinDistributivelyToUnion(
       const SubtreePlan& a, const SubtreePlan& b, const JoinColumns& jcs) const;
 
-  // Extract the join columns from `jcs` that are not the graph.
-  // `leftSideTransitivePath` indicates, if set, that the column indices of the
-  // transitive path can be found on the "left side" of `jcs`. Return
-  // `std::nullopt` if no bind is possible. `TransitivePath` doesn't support all
-  // variants of joins for the bind optimization. If it is supported, return a
-  // pair of indices of the relevant join columns, where the first index
-  // corresponds to the index inside the `TransitivePath` operation, and the
-  // second index to the index of the other operation.
+  // Return a pair of join columns (the first from the transitive path
+  // operation, the second from the other operation with which the result of the
+  // transitive path operation is joined). Otherwise return `std::nullopt`, in
```

Comment:
No need to put this important detail in parentheses. Also "otherwise" is greatly simplifying all the "binding" cases that are (currently) not supported by the transitive path class.